### ISSUE TITLE
Debian control file appears to be malformed

### ISSUE THREAD

**Issue by jasonbcox** [xournalpp/xournalpp#2560]
**Affects versions :**
 - OS: Debian 10 (buster)
 - Version of Xournal++: `1.0.20`
 - Installation method: `xournalpp-1.0.20-Debian-buster-x86_64.deb`

**Describe the bug**
Unable to install the Debian package.

**To Reproduce**
`sudo dpkg --install xournalpp-1.0.20-Debian-buster-x86_64.deb`

**Actual behavior**
```
dpkg: error processing archive xournalpp-1.0.20-Debian-buster-x86_64.deb (--install):
 parsing file '/var/lib/dpkg/tmp.ci/control' near line 3:
 field name 'flexibility,' must be followed by colon
Errors were encountered while processing:
 xournalpp-1.0.20-Debian-buster-x86_64.deb
```

**Expected behavior**
Xournalpp should install successfully.

**Comment by rolandlo** [comment]
`lintian` actually reports this: 
```term
lintian xournalpp-1.0.20-Debian-buster-x86_64.deb 
Skipping xournalpp-1.0.20-Debian-buster-x86_64.deb: syntax error in Cannot parse line 4: flexibility, functionality and speed. Stroke recognizer and other parts are 
```

or with `verbose` option:

```term
lintian -v xournalpp-1.0.20-Debian-buster-x86_64.deb 
N: Using profile ubuntu/main.
Skipping xournalpp-1.0.20-Debian-buster-x86_64.deb: syntax error in Cannot parse line 4: flexibility, functionality and speed. Stroke recognizer and other parts are 

N: No packages selected.
```
Maybe debian doesn't like the whitespaces at the end of line before (and the lines after)?!

**Comment by rolandlo** [comment]
This was fixed in PR #2587

**Comment by hyiltiz** [comment]
Since this is fixed, would it be necessary to update the `1.0.20` package in the Release page? I wanted to upgrade from `1.0.18` to the latest version, and the `1.0.20-hotfix` package size seemed too small it may only be fixing a few files. So I tried to install `1.0.20`, i.e., `1.0.20` is still relevant, esp. considering it is going to be the last release in the `1.0` branch.

**Comment by rolandlo** [comment]
You are supposed to install the `1.0.20-hotfix` package. The low package size only results from stripping the package from debug information. It should be fully functional. See #2602
